src/query_plan/operation.rs

```diff
@@ -704,6 +707,18 @@ pub(crate) mod normalized_inline_fragment_selection {
         pub(crate) selection_id: SelectionId,
     }
 
+    impl NormalizedInlineFragmentData {
+        pub(crate) fn defer_directive_arguments(
+            &self,
+        ) -> Result<Option<DeferDirectiveArguments>, FederationError> {
+            if let Some(directive) = self.directives.get(&name!("defer")) {
```

Comment:
```suggestion
            if let Some(directive) = self.directives.get("defer") {
```

I think this should be equivalent